### PR TITLE
serialize response data

### PR DESCRIPTION
**Pull request by justinwash**

### CODE REVIEW

File: server/src/models/user.rs [hunk 1]
```diff
@@ -1,16 +1,16 @@
 use crate::schema::user;
 use serde::{Deserialize, Serialize};
 
-#[derive(Queryable, Debug, Clone)]
+#[derive(Queryable, Debug, Deserialize, Serialize, Clone)]
```
Comment (justinwash) [inline server/src/models/user.rs:4]: needed to allow json serialization

File: client/src/components/interface/LoginButton.tsx [hunk 2]
```diff
@@ -15,7 +15,7 @@ const LoginButton = () => {
       //testing
       if (API_URL)
         http
-          .post(`${API_URL}/user`, {
+          .put(`${API_URL}/user`, {
```
Comment (afterparty) [inline client/src/components/interface/LoginButton.tsx:18]: WTF ARE YOU DOING

File: server/src/server/http.rs [hunk 3]
```diff
@@ -6,7 +6,10 @@ pub fn gather_routes() -> impl Filter<Extract = impl warp::Reply, Error = warp::
     .map(|name| format!("Hello, {} from the http server!", name))
     .with(warp::cors().allow_any_origin());
 
-  hello.or(add_user()).or(create_session())
+  hello
```
Comment (justinwash) [inline server/src/server/http.rs:9]: is it me you're looking for?

File: client/src/components/interface/LoginButton.tsx (hunk 2 above)
Comment (justinwash) [reply client/src/components/interface/LoginButton.tsx:18]: PUTing

File: server/src/models/user.rs (hunk 1 above)
Comment (afterparty) [reply server/src/models/user.rs:4]: DONT

**Pull request merged**